Source/WebKit/WebProcess/WebPage/CoordinatedGraphics/CompositingCoordinator.cpp

```diff
@@ -170,20 +170,26 @@ bool CompositingCoordinator::flushPendingLayerChanges(OptionSet<FinalizeRenderin
     m_page.updateRendering();
     m_page.flushPendingEditorStateUpdate();
 
+    WTFBeginSignpost(this, FlushRootCompositingLayer);
```

Comment:
I'm not a compositing expert - maybe @magomez could suggest other interessting things to include here. Looks good to me.